ykbuild/Cargo.toml

```diff
@@ -4,7 +4,13 @@ version = "0.1.0"
 authors = ["The Yk Developers"]
 edition = "2021"
 license = "Apache-2.0 OR MIT"
+links = "ykbuild"
```

Comment:
Doesn't this mean ykbuild links with a library called ykbuild? Is that necessary?

Reply:
Not quite, this is a hack so that we can define an `DEP_YKBUILD_YKLLVM` env var in `ykbuild`'s build.rs which is available for any other crate in the yk workspace which depends on it. This then lets other crates know where the install location of the ykllvm bins are.

See https://doc.rust-lang.org/cargo/reference/build-script-examples.html#using-another-sys-crate

Reply:
Can you add a comment explaining this. It is quite surprising at a glance.

Reply:
Sure, how about this 62cc65dd562b90ff64dd544915e1b1554d6559c5?

Reply:
Sure, how about this 62cc65dd562b90ff64dd544915e1b1554d6559c5?